### PR TITLE
[MKT-55]: Feat/update 404 page design

### PR DESCRIPTION
**Pull request by xabg2**
Update 404 page design.
Links added to:
* Drive Home page.
* Help support page.
* Open mail to contact us.(hello@internxt.com)

### CODE REVIEW

File: src/app/i18n/locales/en.json [hunk 1]
```diff
@@ -27,6 +27,26 @@
     },
     "cta": "Get the deal"
   },
+  "notFound": {
+    "header": "Error 404",
+    "title": "Nothing to see here...",
+    "description": "The page you are looking for could not be found.",
+    "card1": {
```
Comment (CandelR) [inline src/app/i18n/locales/en.json:34]: French and Spanish translations are missing

File: src/app/i18n/locales/en.json [hunk 2]
```diff
@@ -27,6 +27,26 @@
...
     "cta": "Get the deal"
   },
+  "notFound": {
+    "header": "Error 404",
+    "title": "Nothing to see here...",
+    "description": "The page you are looking for could not be found.",
+    "card1": {
+      "title": "Return Home",
+      "description": "Go back to Internxt Drive.",
+      "urlRedirect": "https://drive.internxt.com/app"
```
Comment (CandelR) [inline src/app/i18n/locales/en.json]: Not add url to translation files. Use the navigationService in the cards object that has you created to redirect the user to other view of the web, use constants or what do you think fits better depend of the case.

**Review by CandelR — CHANGES_REQUESTED** [review]

File: src/app/i18n/locales/en.json (hunk 1 above)
Comment (xabg2) [reply src/app/i18n/locales/en.json:34]: For the moment it will only be in English because the website has the same behavior.

Comment (xabg2) [reply src/app/i18n/locales/en.json:34]: So, as English is the fallback lang, it always appears in english for the time being.

File: src/app/i18n/locales/en.json (hunk 2 above)
Comment (xabg2) [reply src/app/i18n/locales/en.json]: Fixed

**Review by CandelR — APPROVED** [review]

**Pull request merged**